src/puppetlabs/puppetdb/meta.clj

```diff
@@ -0,0 +1,78 @@
+(ns puppetlabs.puppetdb.meta
+  (:require [clojure.tools.logging :as log]
+            [net.cgrand.moustache :as moustache]
+            [clj-time.core :refer [now]]
+            [puppetlabs.puppetdb.middleware
+             :refer [verify-accepts-json wrap-with-globals
+                     validate-no-query-params wrap-with-puppetdb-middleware]]
+            [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [compojure.core :as compojure]
+            [puppetlabs.kitchensink.core :as kitchensink]
+            [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.meta.version :as v]))
+
+(defn current-version
```

Comment:
yeah, thanks. the docstring is clearly wrong. Do you have a suggested naming convention?